src/ProjectContents/useContentsMutation.tsx

```diff
@@ -34,14 +42,30 @@ export function useUpdateContentMutation({ onSuccess, onError }: Options) {
 
   return useMutation(
     ["content-create"],
-    async ({ token, schemaName }: { token: string; schemaName: string }) => {
-      if (!token || !schemaName) {
-        return;
+    async ({
+      token,
+      schemaName,
+      contentId,
+    }: {
+      token: string;
+      schemaName: string;
+      contentId: string;
+    }) => {
+      if (!content) {
+        return Promise.reject(new Error("Cannot find content data."));
```

Comment:
```suggestion
        throw new Error("Cannot find content data.");
```

Error를 throw하기만 해도 충분할 것 같아용 ㅎㅎ

Reply:
넵!